client/my-sites/plan-features-2023-grid/components/storage-add-on-dropdown.tsx

```diff
@@ -15,7 +15,7 @@ export const StorageAddOnDropdown = ( { planSlug, storageOptions }: StorageAddOn
 	const { setSelectedStorageOptionForPlan } = useDispatch( WpcomPlansUI.store );
 	const selectedStorage = useSelect(
 		( select ) => {
-			return select( WpcomPlansUI.store ).getStorageAddOnForPlan( planSlug );
+			return select( WpcomPlansUI.store ).getSelectedStorageOptionsForPlan( planSlug );
```

Comment:
Is the plural in `getSelectedStorageOptionsForPlan` intentional? i.e. can there be multiple options selected for a given plan (it doesn't look like that's the case)?

Reply:
> Is the plural in getSelectedStorageOptionsForPlan intentional? i.e. can there be multiple options selected for a given plan (it doesn't look like that's the case)?

Absolutely correct -- let me get this fixed. I'll do another once over of the names to make sure I haven't missed anything else.

Reply:
Resolved in https://github.com/Automattic/wp-calypso/pull/80874/commits/c07553728198d56b88bc9c16175149e529ee0844